google/cloud/channel_v1/services/cloud_channel_service/client.py

```diff
@@ -427,8 +427,7 @@ def __init__(
                 client_cert_source_for_mtls=client_cert_source_func,
                 quota_project_id=client_options.quota_project_id,
                 client_info=client_info,
-                always_use_jwt_access=not credentials
-                and (
+                always_use_jwt_access=(
```

Comment:
This change will revert the changes in https://github.com/googleapis/python-channel/pull/86. I think we need to keep the original line.

Reply:
I've reverted the change to unblock this PR.